packages/flutter_adaptive_scaffold/lib/src/breakpoints.dart

```diff
@@ -117,6 +117,15 @@ class Breakpoint {
     this.andUp = false,
   });
 
+  /// Returns a [Breakpoint] that can be used as a fallthrough in the
+  /// case that no other breakpoint is active.
+  const Breakpoint.standard({this.platform})
```

Comment:
| width x height | breakpoint by width | breakpoint by height | active breakpoint |
|-|-|-|-|
| 360.0 x 672.0 | small | medium | small |
| **672.0 x 360.0** | medium | small | standard |
| 387.4 x 877.1 | small | medium | small |
| **877.1 x 387.4** | mediumLarge | small | standard |

In the sizes written in bold above, shouldn't the breakpoint calculated by the width be used instead of `standard`? If those sizes fall into that breakpoint, does it mean we always need to specify `Breakpoint.standard`? It is tedious and inconvenient.

I think it is a regression introduced by #7300. We didn't have to care about the standard breakpoint before the change.

Reply:
@kaboc i see what you mean now. I think that the bug i fixed in here originally was also a valid bug but now i get what you mean too.

I've added tests for both cases and implemented a fix. Can you check?

Reply:
The code sample I posted in the issue page and my actual app that had this issue are both working correctly now. Thank you!